shared/schema.gql

```diff
@@ -340,6 +351,7 @@ extend type Subscription {
 ###############################################################################
 
 directive @canAccessSubscriptionOffer on FIELD_DEFINITION
+directive @canManageSubscription on FIELD_DEFINITION
```

Comment:
I assume the changes starting here and below, are related to something else, otherwise I don't understand it 😄